File: EasyPack.Domain/model/Carga.cs
using System;

namespace EasyPack.Domain.model
{
    public class Carga
    {
        public Carga()
        {

        }

        public int Id { get; set; }
        public double Peso { get; set; }
        public double Altura { get; set; }
        public double Comprimento { get; set; }
        public double Largura { get; set; }
        public DateTime Data_Entrega { get; set; }
        public int EntregaId { get; set; }
        public Entrega Entrega { get; set; }


    }
}

File: Front/EasyPack-App/src/app/_models/Carga.ts
import { Entrega } from "./Entrega";

export interface Carga {

  id: number;
  peso: number;
  altura: number;
  comprimento: number;
  largura: number;
  data_Entrega: Date;
  entregaId: number;
  entrega: Entrega;
}
